aspnetcore/security/authentication/cookie.md

```diff
@@ -1,72 +1,43 @@
 ---
 title: Use cookie authentication without ASP.NET Core Identity
 author: rick-anderson
-description: An explanation of using cookie authentication without ASP.NET Core Identity
+description: Learn how to use cookie authentication without ASP.NET Core Identity.
+monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
-ms.date: 02/25/2019
+ms.date: 07/06/2019
 uid: security/authentication/cookie
 ---
 # Use cookie authentication without ASP.NET Core Identity
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Luke Latham](https://github.com/guardrex)
 
-As you've seen in the earlier authentication topics, [ASP.NET Core Identity](xref:security/authentication/identity) is a complete, full-featured authentication provider for creating and maintaining logins. However, you may want to use your own custom authentication logic with cookie-based authentication at times. You can use cookie-based authentication as a standalone authentication provider without ASP.NET Core Identity.
+[ASP.NET Core Identity](xref:security/authentication/identity) is a complete, full-featured authentication provider for creating and maintaining logins. However, you can use cookie-based authentication as a standalone authentication provider without ASP.NET Core Identity.
```

Comment:
[ASP.NET Core Identity](xref:security/authentication/identity) is a complete, full-featured authentication provider for creating and maintaining logins. However, a cookie-based authentication authentication provider without ASP.NET Core Identity can be used.

Reply:
For more information, see [Use cookie authentication without ASP.NET Core Identity](security/authentication/cookie).